goal_src/jak2/engine/debug/viewer.gc

```diff
@@ -5,5 +5,262 @@
 ;; name in dgo: viewer
 ;; dgos: ENGINE, GAME
 
+(declare-type viewer process-drawable)
+(define-extern *viewer* viewer)
+
 ;; DECOMP BEGINS
 
+(define *viewer-sg* (new 'static 'skeleton-group
+                      :bounds (new 'static 'vector :w 16384.0)
+                      :lod-dist (new 'static 'array float 6 4095996000.0 0.0 0.0 0.0 0.0 0.0)
```

Comment:
I think the skelgroup stuff is merged - does this get converted to `defskelgroup` on the latest master?

Reply:
I checked and it doesn't - this seems to be a case where the static data is completely empty besides the `bounds` and `lod-dist` and the only thing that is defined in the `top-level` is the `skeleton-group` itself, there is no `let`, so the decompiler doesn't try to rewrite it.
The IR for Jak 1 looks exactly the same.